docs/user/using-an-OIDC-provider.md

```diff
@@ -241,3 +241,59 @@ spec:
         path: /
 EOF
 ```
+
+## Debugging auth failures when using OIDC
+
+If you find after configuring your OIDC/OAuth2 setup following the above instructions, that although you can successfully authenticate with your provider you are nonetheless unable to login to Kubeapps but instead see a 403 or 401 request in the browser's debugger, then you will need to investigate *why* the Kubernetes cluster is not accepting your credential.
+
+### Viewing the JWT id token
+
+The easiest way to check the credential that is being used is to temporarily set the `--set-authorization-header=true` option for oauth2 proxy of the kubeapps deployment. Edit the `kubeapps` deployment in the specific namespace on your cluster and add the option to the command for the oauth2 proxy container. Once the deployment runs a new container with the extra option, Kubeapps will then include the `id_token` that is being used to authenticate you with the Kubernetes api server in the response back to the browser.
```

Comment:
> Edit the `kubeapps` deployment

People often get confused with the deployment names. I would specify to ignore the ones called `kubeapps-internal-*`. Also, if you can include a YAML snippet with an example of an added flag I think it will be much appreciated.

Reply:
Thanks, yep, I've removed that now in place of a copy-n-pastable patch.